core/tests/test_account/src/lib.rs

```diff
@@ -191,6 +195,52 @@ impl Account {
         .unwrap()
     }
 
+    pub fn get_l1_prank_tx(
```

Comment:
I understand that here we use the same vocabulary as in foundry, but I am not sure that it will be understandable at a first glance for everyone in the platform team, please rename to a more understandable name